crates/ggml/src/lib.rs

```diff
@@ -29,6 +29,65 @@ mod tests;
 #[cfg(feature = "metal")]
 pub mod metal;
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+///Accelerators supported by `ggml`.
```

Comment:
Very minor style nit: `/// ` with a space after the slashes, but not a huge issue. (I'll see if rustfmt has a way to do that automatically.)